package.json

```diff
@@ -29,7 +29,7 @@
     "babel": "6.5.2",
     "babel-core": "6.18.2",
     "babel-loader": "6.2.7",
-    "babel-plugin-jsm-to-esmodules": "0.2.0",
+    "babel-plugin-jsm-to-commonjs": "0.1.0",
```

Comment:
I'd suggest adding an XXX comment saying "let's get rid of this when we stop supporting Node 4", or some such.